.github/workflows/pull-request.yml

```diff
@@ -2,13 +2,19 @@ name: Build
 on:
   pull_request:
     branches: [ main ]
-env:
-  DIRECTORY: ./app
+defaults:
+  run:
+    working-directory: app
```

Comment:
replaced `DIRECTORY: ./app` with this. what are your thoughts?
`docker build $DIRECTORY` now becomes `docker build .`

Reply:
very nice